libc/config/linux/x86_64/entrypoints.txt

```diff
@@ -594,7 +600,9 @@ if(LIBC_TYPES_HAS_FLOAT16)
   if(LIBC_TYPES_HAS_FLOAT128)
     list(APPEND TARGET_LIBM_ENTRYPOINTS
       # math.h C23 mixed _Float16 and _Float128 entrypoints
+      libc.src.math.f16addf128
       libc.src.math.f16fmaf128
+      libc.src.math.f16subf128
     )
```

Comment:
I haven't enabled these on AArch64 because I think they will cause the same Clang 11 backend error (failure to select instructions for `fp_round` from `f128` to `f16`) as in #96952 due to this code:

https://github.com/llvm/llvm-project/pull/97072/files#diff-9916523646ef256733d16b860bb59b31d4cc2448632c871b6b2256fddadf685bR104-R115